src/test/config/test-config.ts

```diff
@@ -13,6 +13,7 @@ function mergeAll<T>(objects: Partial<T>[]): T {
 }
 
 export function createTestConfig(config?: IUnleashOptions): IUnleashConfig {
+    getLogger.setMuteError(true);
```

Comment:
I wonder what's this, and whether it should be included in this PR.

Reply:
It's muting the error logger of our test outputs, so we don't end up with tons of error logging that isn't actually an error when running tests.